test/unit/tracks/text-tracks.test.js

```diff
@@ -365,6 +366,12 @@ QUnit.test('removes cuechange event when text track is hidden for emulated track
     mode: 'showing'
   });
 
+  const oldVttjs = window.vttjs;
```

Comment:
this change probably should be removed from this PR in favor of https://github.com/videojs/video.js/pull/4148/files#diff-45ae6eaab623b37642b347fe7707682aR338